notebooks/signac_106_Ideal_Gas_Example.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>It might be nice to show the labels or operations we created above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>+1</p>

Reply:
<p>This is a bug, in two ways, and it's a little crazy. First, the signac-flow syntax of this notebook is wayyy old. This doesn't use the FlowProject correctly and needs to be rewritten. As a result of the outdated/invalid syntax, the FlowProject that is created has zero registered operations. However, this also pointed out to me that the <code>print_status</code>  function does not work correctly for FlowProjects with no operations. Alyssa and I noticed this, and I fixed it in this PR: https://github.com/glotzerlab/signac-flow/pull/460</p><p> </p><p>Fixing the root cause of the problem requires rewriting the notebook to use modern signac-flow features. Alyssa added a warning to denote the syntax is outdated and we ought to prioritize updating this specific notebook in a near-term follow-up PR.</p>